daphne/src/vdaf/prio3_test.rs

```diff
@@ -76,6 +76,7 @@ fn test_prepare(
     // Unshard
     let agg_res = prio3_unshard(
         config,
+        1,
```

Comment:
I'm probably misunderstanding something here -- shouldn't this be `2`?

Reply:
This is the "report count". Only one report is aggregated in this test.